frontend/utilities/_print.scss

```diff
@@ -48,6 +48,10 @@
     color: govuk-colour('black')
   }
 
+  .govuk-\!-font-size-16 {
```

Comment:
Not related, but small formatting bug I noticed when checking print views. Because we drop font sizes in print anything using this font-size becomes proportionally too large.